packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -270,7 +299,14 @@ export class ChildLogger extends TelemetryLogger {
     public static create(
         baseLogger?: ITelemetryBaseLogger,
         namespace?: string,
-        properties?: ITelemetryLoggerPropertyBags): TelemetryLogger {
+        properties: ITelemetryLoggerPropertyBags = {},
+        globalProperties: ITelemetryLoggerPropertyBags = {},
+        layerVersion?: string,
+    ): TelemetryLogger {
+        const namedLayerVersion = namespace !== undefined && layerVersion !== undefined
+            ? `${namespace}:${layerVersion}`
+            : undefined;
+
         // if we are creating a child of a child, rather than nest, which will increase
         // the callstack overhead, just generate a new logger that includes everything from the previous
         if (baseLogger instanceof ChildLogger) {
```

Comment:
this doesn't always work across our layers, loader/runtime. it wasn't a problem before as this code is really just an optimization as we ran into perf problems with extereme nesting, but in this case it will be an issue, as global props won't cross this boundary

Reply:
Another idea is to build this directly into the telemetry logger class with a function like `protected addGlobalProps?()` the advantage here is that is would also work with the multisink logger, so we could get the logs into the debug logger. 

Since we don't know if the parent will support globalProps, we should continue to set these props locally. the locall props should override the global props as in the case they both exist it won't matter as the values should match, if they don't, the lowest logger should win

Reply:
Ahhh of course.  Thanks, I'll think it over